build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -11,7 +11,7 @@
   <ItemGroup>
     <PublishOutputExtensions Include="$(ExeExtension);.dll;.pdb;.deps.json;.runtimeconfig.json" />
     <FilesToMove Include="$(BinaryToCorehostifyRelDir)/csc.exe;$(BinaryToCorehostifyRelDir)/MSBuild.exe;" />
-    <BundledTools Include="csc;MSBuild;NuGet.CommandLine.XPlat;dotnet" />
+    <BundledTools Include="csc;MSBuild;NuGet.CommandLine.XPlat;dotnet;vstest.console;testhost;datacollector" />
```

Comment:
Yes we need all these tools. vstest.console is calling testhost and datacollector.